flink-end-to-end-tests/run-pre-commit-tests.sh

```diff
@@ -60,6 +60,7 @@ run_test "Modern Kafka end-to-end test" "$END_TO_END_DIR/test-scripts/test_strea
 run_test "Kinesis end-to-end test" "$END_TO_END_DIR/test-scripts/test_streaming_kinesis.sh"
 run_test "class loading end-to-end test" "$END_TO_END_DIR/test-scripts/test_streaming_classloader.sh"
 run_test "Distributed cache end-to-end test" "$END_TO_END_DIR/test-scripts/test_streaming_distributed_cache_via_blob.sh"
+run_test "Dependency shading of table module test" "$END_TO_END_DIR/test-scripts/test_table_shaded_dependencies.sh"
```

Comment:
We remove this again while merging, right?

Reply:
Do you mean you wouldn't add this test to pre commit tests?

Reply:
I think the pre commit tests should just test the very core functionality. It is also ok, to wait a night for getting noticed about an violation.